src/utils/smartWallet.js

```diff
@@ -91,6 +91,16 @@ export const userHasSmartWallet = (accounts: Accounts = []): boolean => {
   return accounts.some(acc => acc.type === ACCOUNT_TYPES.SMART_WALLET);
 };
 
+export const getPreferredWalletId = (accounts: Accounts = []): string => {
+  const smartWallet = accounts.find(({ type }) =>
```

Comment:
How about creating a new helper `findFirstSmartAccount` ?
I had the same need for this method in another task, so I think it's worth having it